pinot-common/src/main/java/org/apache/pinot/common/utils/ZkStarter.java

```diff
@@ -34,16 +35,22 @@
 public class ZkStarter {
   private static final Logger LOGGER = LoggerFactory.getLogger(ZkStarter.class);
   public static final int DEFAULT_ZK_TEST_PORT = 2191;
-  public static final String DEFAULT_ZK_STR = "localhost:" + DEFAULT_ZK_TEST_PORT;
   private static final int DEFAULT_ZK_CLIENT_RETRIES = 10;
+  private static int ZK_TEST_PORT = 2191;
```

Comment:
Isn't it duplicate with Line 37? If not, what's the purpose of having two ZK test ports?

Reply:
+1

Reply:
done